sdk/src/dlob/DLOB.ts

```diff
@@ -517,17 +452,9 @@ export class DLOB {
 			);
 
 			for (const makerNode of makerNodeGenerator) {
-				const bidUserAuthority = this.userMap.getUserAuthority(
-					makerNode.userAccount.toString()
-				);
-				const askUserAuthority = this.userMap.getUserAuthority(
-					takerNode.userAccount.toString()
-				);
-
 				// Can't match orders from the same authority
-				const sameAuthority = bidUserAuthority.equals(askUserAuthority);
-
-				if (sameAuthority) {
+				const sameUser = takerNode.userAccount.equals(makerNode.userAccount);
```

Comment:
this isn't checking for same authority anymore, is it intended? if so should remove the comment above

Reply:
yes it's intended to remove dependency on user map